packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.scss

```diff
@@ -60,6 +60,7 @@
   max-width: 364px;
   border: 0;
   box-shadow: none !important;
+  width: calc(100% + 1px);
```

Comment:
I'm confused why this is needed. Why not `white-space: nowrap;`

Reply:
`white-space: nowrap` isn't desirable because we still want content to wrap.

<img width="404" alt="screen shot 2018-06-06 at 6 49 19 pm" src="https://user-images.githubusercontent.com/1291968/41073975-9f1a7a36-69ba-11e8-9c00-3bbf9fcfa00f.png">